README.md

```diff
@@ -60,6 +60,7 @@ B.sub(A); // returns [[2, 2, 2], [2, 2, 2], [2, 2, 2]]
 ```javascript
 A.mul(B); // returns [[3, 8, 15], [24, 35, 48], [56, 80, 99]]
 ```
+Nota Bene: This is not classical matrix multiplication (which is implemented using the prod() method). This simply multiplies together each element in matrix A with the corresponding element in matrix B. If A and B are not the same size, it will produce some NaN results.
```

Comment:
@NotoriousWebmaster Can you change it to **NOTE** ?

Reply:
Yes, done.